RoboFile.php

```diff
@@ -152,7 +153,10 @@ private function postProcessModuleDocFile($documentationFile, $name, $source)
         foreach (['3.1', '2.5', '1.8'] as $branch) {
             $buttons[$branch] = self::REPO_BLOB_URL."/$branch/docs/modules/$name.md";
         }
+
         $buttonHtml = "\n\n".'<div class="btn-group" role="group" style="float: right" aria-label="...">';
+        $releasesUrl = "https://github.com/Codeception/module-$name/releases";
```

Comment:
It would be a bit better to pass $repositoryName from the previous function, but this works too.
Example:
```php
$name =  'ZendExpressive';
$repositoryName = 'module-zendexpressive';
```
Actually $repositoryname is the second part of package name, but it matches repository name in all cases.

Reply:
ZendExpressive link works. I see no problem here